core/whistle-1.0.0/src/api/wapi_presence.erl

```diff
@@ -108,19 +108,19 @@
 %% MWI Update
 -define(MWI_REQ_HEADERS, [<<"To">>
                           ,<<"Messages-New">>
-                          ,<<"Messages-Waiting">>
+                          ,<<"Messages-Saved">>
                          ]).
 -define(OPTIONAL_MWI_REQ_HEADERS, [<<"Messages-Urgent">>
-                                   ,<<"Messages-Urgent-Waiting">>
+                                   ,<<"Messages-Urgent-Saved">>
                                    ,<<"Call-ID">>
                                   ]).
 -define(MWI_REQ_VALUES, [{<<"Event-Category">>, <<"presence">>}
                          ,{<<"Event-Name">>, <<"mwi_update">>}
                         ]).
 -define(MWI_REQ_TYPES, [{<<"Messages-New">>, fun(I) -> is_integer(wh_util:to_integer(I)) end}
-                        ,{<<"Messages-Waiting">>, fun(I) -> is_integer(wh_util:to_integer(I)) end}
+                        ,{<<"Messages-Saved">>, fun(I) -> is_integer(wh_util:to_integer(I)) end}
```

Comment:
Personally, I think casting to integer first should be removed. The caller should put integers there! I know it was there already, but reading it now and seeing your changes elsewhere, I think just a straight `{<<"Messages-Saved">>, fun is_integer/1}` is sufficient. Thoughts?